cmd/juju/charmhub/convert.go

```diff
@@ -139,7 +140,7 @@ func channelSeries(platforms []corecharm.Platform) set.Strings {
 func channelBases(platforms []corecharm.Platform) set.Strings {
 	bases := set.NewStrings()
 	for _, v := range platforms {
-		bases.Add(fmt.Sprintf("%s/%s", v.OS, v.Channel))
+		bases.Add(fmt.Sprintf("%s:%s", v.OS, v.Channel))
```

Comment:
I'm including this change in here as well. Should merge cleanly when https://github.com/juju/juju/commit/3a279999adbe6b21d6719540e36df4066b3dee1f#diff-2e6e3fd3f525cab09fb3666efde3b00ed439ce2d1bb583d14b9f307d96b12c54 is merged into develop.